lib/components/editable_select_component.dart

```diff
@@ -113,6 +113,7 @@ class EditableSelectComponent extends SelectComponent {
   }
 
   get current_input_value => findPart("display_input").value;
+  get name => findPart("input").name;
```

Comment:
There are two inputs: "display_input" and "input". This code selects input with exactly "input" name.